cli/src/commands/start.rs

```diff
@@ -634,21 +641,25 @@ impl Start {
                 if let Some(rest_ip) = rest_ip {
                     println!("🌐 Starting the REST server at {}.\n", rest_ip.to_string().bold());
 
-                    let jwt_secret = if let Some(jwt_b64) = &self.jwt_secret {
-                        if self.jwt_timestamp.is_none() {
-                            bail!("The '--jwt-timestamp' flag must be set if the '--jwt-secret' flag is set");
-                        }
-                        let jwt_bytes = BASE64_STANDARD.decode(jwt_b64).map_err(|_| anyhow::anyhow!("Invalid JWT secret"))?;
-                        if jwt_bytes.len() != 16 {
-                            bail!("The JWT secret must be 16 bytes long");
+                    if !self.nojwt {
+                        let jwt_secret = if let Some(jwt_b64) = &self.jwt_secret {
+                            if self.jwt_timestamp.is_none() {
+                                bail!("The '--jwt-timestamp' flag must be set if the '--jwt-secret' flag is set");
+                            }
+                            let jwt_bytes = BASE64_STANDARD.decode(jwt_b64).map_err(|_| anyhow::anyhow!("Invalid JWT secret"))?;
+                            if jwt_bytes.len() != 16 {
+                                bail!("The JWT secret must be 16 bytes long");
+                            }
+                            Some(jwt_bytes)
+                        } else {
+                            None
+                        };
+
+                        if let Ok(jwt_token) = snarkos_node_rest::Claims::new(account.address(), jwt_secret, self.jwt_timestamp).to_jwt_string() {
+                            println!("🔑 Your one-time JWT token is {}\n", jwt_token.dimmed());
+                            // Store the JWT secret to a file.
+                            std::fs::write(format!("jwt_secret_{}.txt", account.address()), jwt_token)?;
```

Comment:
hmm perhaps we should write it to the ledger folder, so that it's at a predictable location? or at least notify the user where it is being stored